### ISSUE TITLE
Syntax Error

### ISSUE THREAD

**Issue by 1150AA** [davidohne/ConCamEnabler#1]
Expected "," or "}" but found ":"

Any ideas where the error is?

**Comment by davidohne** [comment]
Did you try to run the concam script in "Script Editor"? It is still working on Ventura. Can't test Sonoma.
![concom](https://github.com/davidohne/ConCamEnabler/assets/723166/731ad334-9777-4348-8152-66f8951fad63)

**Comment by 1150AA** [comment]
Yes running in Script Editor. I am unfortunately on Sonoma though so that may be the issue.
